packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -144,16 +148,23 @@ export class DefaultCrudRepository<
     const model = dataSource.getModel(definition.name);
     if (model) {
       // The backing persisted model has been already defined.
-      return model as typeof juggler.PersistedModel & ObserverMixin;
+      // Will just need juggler.PersistedModelClass once https://github.com/strongloop/loopback-datasource-juggler/issues/1824 is addressed
+      return model as typeof juggler.PersistedModel & juggler.ObserverMixin;
     }
 
     return this.definePersistedModel(entityClass);
   }
 
-  // Create an internal legacy Model attached to the datasource
+  /**
+   * Create a model class and attach it to the datasource. This method can be
+   * overriden in sub-classes to acess methods and properties in the generated
+   * model class.
+   * @param entityClass - Legacy entity class
```

Comment:
This is not correct. `entityClass` is the LB4 `Entity` constructor, e.g. `Product` or `User` inheriting from LB4 `Entity`.

The function returns the legacy persisted model class (e.g. `Product` inheriting from `juggler.PersistedModel`).

Please update this doc accordingly.

Reply:
Updated to:
```ts
  /**
   * Creates a legacy persisted model class, attaches it to the datasource and
   * returns it. This method can be overriden in sub-classes to acess methods
   * and properties in the generated model class.
   * @param entityClass - LB4 Entity constructor
   */
```